shelley/chain-and-ledger/executable-spec/test/STSTests.hs

```diff
@@ -8,8 +8,8 @@ import qualified Data.Map.Strict as Map (empty, singleton)
 import           Test.Tasty (TestTree, testGroup)
 import           Test.Tasty.HUnit (Assertion, assertBool, testCase, (@?=))
 
-import           Examples (CHAINExample (..), alicePay, bobPay, carlPay, dariaPay, ex1, ex2, ex3,
-                     ex4, ex5, ex6, ex7)
+import           Examples (CHAINExample (..), alicePay, bobPay, carlPay, dariaPay, ex1, ex10, ex2,
```

Comment:
can stylish sort correctly here, i.e., ex1, ex2 ... ex10 ?

Reply:
I don't know, I saw that and did not love it. Shortlex order would be much better!